data_sources/github_webhooks.yml

```diff
@@ -1,15 +1,12 @@
-name: GitHub
```

Comment:
@P4T12ICK : This data source `GitHub Webhooks`  is not used by any detection, is there a reason to add this? I think better to delete it or update this for the deprecate content since the whole data source and ingesting is clearer that way

Reply:
I changed the old data source to github webhooks. Will update the old deprecated one with these.